recipemd_extract/main.py

```diff
@@ -59,8 +62,9 @@ def main():
 
 	recipe=extract(url,args.debug)
 
-	if(recipe):
-		writeRecipe(recipe, file)
+	if isinstance(recipe,Recipe):
+		result = writeRecipe(recipe, file)
+		print('Recipe downloaded to ' + result)
```

Comment:
Just a minor thing: I'd prefer "Recipe written to" instead of "downloaded", as the recipe is not directly downloaded, but converted on the way.